README.md

```diff
@@ -126,6 +126,10 @@ beyond the LTS release.
 </tr>
 </table>
 
+## A note about v4.2 LTS
+
+While v4.2 was released in October, *this is a one-off event!* There was no LTS release plan in place in April 2015; v4.2 LTS was released in August in order to get v4.x updates into an LTS release as quickly as possible. In the future **LTS releases will be cut in April**.
```

Comment:
Ah thanks. Is there a term for "a release scheduled to be LTS" vs "6 month and done" release?

Is this better:

> Releases versions that will ultimately receive long term support are **even numbered** and cut in **April** (they are considered "latest/stable" 'til October when they go into LTS mode).

? I'm trying to make it more clear that if you want to get the newest thing that will ultimately recieve LTS (put differently, if you want to get the maximum possible period of support), those are the major versions released in **April**. It's also technically correct as you say that LTS designations happen in October, but saying "LTS versions are cut in October" is **extremely** likely to cause confusion as that also when short/Odd releases happen.